### DISCUSSION TITLE
Does adonis v5 have anything like repl

### DISCUSSION THREAD

**Discussion by PieterVenter7** [adonisjs/core#1374] [Help] [answered]
Or is there a way to implement it

**Comment by thetutlage** [comment, marked as answer]
Not yet. Coz repl needs a way to transpile Typescript code to Javascript on the fly. This is doable, but needs some great effort to implement it properly.

If you or anyone has experience with tools like `ts-node` or `esbuild`, then I am happy to advance this conversation and decide together on how to implement it properly

**Reply by dcrystalj** [reply]
can you setup repl on compiled javascript  ?
